src/components/LanguageSelector/LanguageSelector.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div>
+    <template v-for="{ locale, name } in languages">
+      <NuxtLink
+        v-if="locale !== $i18n.locale"
+        :key="locale"
+        :to="getLocalePath(locale)"
+        :hreflang="locale"
+        class="app-menu__link"
+        @click="$emit('close')"
+      >
+        <SvgIcon name="world-icon" class="app-menu__icon" />
+        <span class="app-menu__link-name">
+          {{ name }}
+        </span>
+      </NuxtLink>
+    </template>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { storeToRefs } from "pinia";
+import { useStore } from "~/stores/store";
+import { useHistoryStore } from "~/stores/history";
+
+const languages = [
+  { locale: "en", name: "english" },
```

Comment:
"English" and "Nederlands" should be capitalised.

Reply:
It's currently handled in CSS 

```
.app-menu__link-name::first-letter {
  text-transform: uppercase;
}
```

I don't have preferences here, but we maybe might keep it outside of this PR :)

Reply:
Yes, we can keep it outside of the PR. As a rule: markup and content should work standalone as HTML without CSS. Screen readers, search bots etc may not take CSS into account.